components/layout/Page.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Remove this. we don't use this anymore in React

Reply:
Ok!